### PR TITLE
PBM-1430 Added about a docs page

### PR DESCRIPTION
**Pull request by nastena1606**
Documented the message that docs reflect the latest version

new file:   docs/about-docs.md
	modified:   docs/copyright.md
	modified:   docs/index.md
	modified:   mkdocs-base.yml

### CODE REVIEW

File: docs/about-docs.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# About Percona Backup for MongoDB documentation
+
+In this documentation you will find comprehensive information about the features, operations, and functionality of Percona Backup for MongoDB (PBM). Also reference materials on PBM commands and configuration file options are available for you here.
+
+## Versions
+
+We encourage our users to run the latest version of PBM so that they receive all bug fixes and can enjoy all new features and improvements available with the latest version. For this reason, the documentation always reflects the latest PBM version.
```
Comment (radoslawszulgo) [inline docs/about-docs.md:7]:
```suggestion
You should run the latest version of PBM to receive all bug and security fixes and benefit from the new features. For this reason, the documentation always reflects the latest PBM version. 
```

File: docs/about-docs.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# About Percona Backup for MongoDB documentation
+
+In this documentation you will find comprehensive information about the features, operations, and functionality of Percona Backup for MongoDB (PBM). Also reference materials on PBM commands and configuration file options are available for you here.
+
+## Versions
+
+We encourage our users to run the latest version of PBM so that they receive all bug fixes and can enjoy all new features and improvements available with the latest version. For this reason, the documentation always reflects the latest PBM version. 
+
+To track the changes in features, we indicate the version where a feature appeared or a change was introduced. Check the release notes for detailed information.
```
Comment (radoslawszulgo) [inline docs/about-docs.md:9]:
```suggestion
To track feature changes, an additional version indication is added where a feature appeared or a change was introduced. Please take a look at the release notes for detailed information.
```

File: docs/about-docs.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+## Report an issue or request a change
+
+To report an issue with the documentation, leave your feedback using the widget at the bottom of the page or open a ticket in [Percona Backup for MongoDB Jira project :octicons-link-external-16:](https://perconadev.atlassian.net/jira/software/c/projects/PBM/issues).
+
+## How to contribute
+
+Thank you for willing to contribute to Percona Backup for MongoDB project. Please check the [Contribution guidelines](reference/contributig.md) for  instructions.
```
Comment (radoslawszulgo) [inline docs/about-docs.md:17]:
```suggestion
Thank you for your willingness to contribute to Percona Backup for MongoDB project. Please check the [Contribution guidelines](reference/contributing.md) for instructions.
```

File: docs/copyright.md [hunk 4]
```diff
@@ -2,5 +2,5 @@
 
 ## Documentation licensing
 
-Percona Backup for MongoDB documentation is (C)2016-2023 Percona LLC and/or its affiliates
+Percona Backup for MongoDB documentation is (C)2016-2024 Percona LLC and/or its affiliates
```
Comment (radoslawszulgo) [inline docs/copyright.md:5]: Can we use a variable here to avoid hardcoded dates in all places?

File: docs/index.md [hunk 5]
```diff
@@ -1,19 +1,20 @@
 # Percona Backup for MongoDB documentation
 
-Percona Backup for MongoDB (PBM) is an open source and distributed solution for consistent backups and restores of [MongoDB sharded clusters and replica sets](details/deployments.md) to a specific point in time. Check [supported MongoDB versions](details/versions.md).
+!!! note ""
+
+    This is the documentation for the latest release, **PBM {{release}}** ([Release Notes](release-notes/{{release}}.md)).
+
+Percona Backup for MongoDB (PBM) is an open source and distributed solution for consistent backups and restores of [MongoDB sharded clusters and replica sets](details/deployments.md) to a specific point in time.
```
Comment (radoslawszulgo) [inline docs/index.md:7]:
```suggestion
Percona Backup for MongoDB (PBM) is an open-source and distributed solution for consistent backups and restores of [MongoDB sharded clusters and replica sets](details/deployments.md) to a specific point in time. 
```

**Review by radoslawszulgo — CHANGES_REQUESTED** [review]

File: docs/about-docs.md (hunk 3 above)
Comment (radoslawszulgo) [reply docs/about-docs.md:17]: + we need to rename the "contributig.md" file ;)

File: docs/copyright.md (hunk 4 above)
Comment (nastena1606) [reply docs/copyright.md]: Done

**Review by radoslawszulgo — APPROVED** [review]

**Pull request merged**